### PR TITLE
Test-for-Lesson-video

### PR DESCRIPTION
**Pull request by ngovannghiaframgia**
ready
Screenshot from 2019-03-11 14-30-15

### CODE REVIEW

**Comment by ngovannghiaframgia** [comment]
ready

**Comment by ngovannghiaframgia** [comment]
ready
Screenshot from 2019-03-11 14-38-00

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -11,6 +11,9 @@
 // about supported directives.
 //
 //= require "jquery.min.js"
+//= require jquery
+//= require jquery_ujs
+//= require rails-ujs
```
Comment (huongnguyenmta) [inline app/assets/javascripts/application.js]: ở dưới require rồi mà em ...

**Comment by ngovannghiaframgia** [comment]
ready

**Comment by ngovannghiaframgia** [comment]
ready
Screenshot from 2019-03-11 15-06-39

**Comment by ngovannghiaframgia** [comment]
ready

**Pull request merged**